viewmaster/viewmaster.py

```
@@ -1227,11 +1381,11 @@ def get_product_page_info(query_pdsfile):
 
     # Get neighbor navigation and warn about timing if it is very slow
     start_time = datetime.datetime.now()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)